libs/utils/decorate.js

```diff
@@ -331,20 +408,83 @@ export async function loadCDT(el, classList) {
   }
 }
 
+export function isAccessible(anchorTag) {
```

Comment:
I still have a general naming issue with the methods in this file. The file itself tailors to multiple types of content, including videos. `isAccessible` is relevant just for videos, but it could very well suggest it can apply to any other blocks. I'd say we need to better separate video logic as such through some better namings.